tests/integration/test_charm.py

```diff
@@ -16,6 +16,7 @@
 from botocore.config import Config
 from ops.model import ActiveStatus, Application
 from pytest_operator.plugin import Model
+from saml_test_helper import SamlK8sTestHelper  # pylint: disable=import-error
```

Comment:
Why doesn't pylint like this?

Reply:
This is the failure on the linter: tests/integration/test_charm.py:19:0: E0401: Unable to import 'saml_test_helper' (import-error)